stable/spark/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: spark
-version: 0.2.1
+version: 0.2.2
```

Comment:
This might warrant a big version jump

Reply:
Yes it does. All stable charts should be at least `1.0.0` (See [`Chart review guidelines` > `Versioning`](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#versioning)).

This PR should also add an `Upgrading` section to the README to describe any manual steps necessary to upgrade to the new `1.0.0`.

Reply:
Thanks for the feedback @scottrigby . Since there is no prior major version, I'll bump this 1.0.0 and update the README